### ISSUE TITLE
Certificates for an organisation with a long  name do not  work

### ISSUE THREAD

**Issue by klvries** [OSGP/Platform#122]
Cedrtificates for an organisation with a long  name do not  work

It seems that the code has a maximum length the name of the certificate.

Eg the name GemeenteSudwestFryslan does not work correct

whether SudwestFryslan works correct

In the sofware there are no limitations for the length of the name.

S

**Comment by rlemmers** [comment]
"In the software there are no limitations for the length of the name.": there should be no limitations, but it's very well possible that a name is cut off in some place.

**Comment by kevinsmeets** [comment]
I have tried to reproduce this situation. This is what I did:

- I took the certificate for GemeenteSudwestFryslan 
- I created an organization called GemeenteSudwestFryslan 
- I created a user for the organization

Then I proceeded with the following:

- I logged in to web-operator
- I logged in to web-owner
- I logged in to web-net-management

Everything works as expected and there is no problem with the certificate, the organization name or the length of the organization name. A certificate can hold a CN (which is equal to the organization name) up to 256 characters in length, GemeenteSudwestFryslan is only 22 characters.

Therefore, I have to conclude there is nothing to do here: there is no problem or bug in the security logic.
